build_tools/bazel/iree.bazelrc

```diff
@@ -18,6 +18,9 @@ common --experimental_repo_remote_exec
 # Actually printing output on errors is... a useful default
 test --test_output=errors
 
+build --experimental_cc_shared_library?
```

Comment:
is the question mark intended here?

Also maybe move this up next to https://github.com/google/iree/blob/95eb1e5a9fe7a0e8c67e5b42df4b5b4ebeef5f4f/build_tools/bazel/iree.bazelrc#L16-L17

and add a similar comment. Each setting like this should have some context in a comment so we know roughly why the flag was added. That makes it easier to make changes safely if circumstances change (e.g. if TensorFlow drops the flag or if it is no longer experimental after a certain Bazel version)